### PR TITLE
Add initial project guide

### PR DESCRIPTION
**Pull request by bensleveritt**
Adds a project guide for Svelte, and updates the docs to link to it

### CODE REVIEW

**Review by trishalim — APPROVED** [review]

File: homepage/homepage/app/docs/[framework]/[...slug]/project-setup/svelte.mdx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+<CodeGroup>
+```ts
+// src/lib/schema.ts
+import { Account, Profile, co } from "jazz-tools";
+
+export class MyProfile extends Profile {
+  name = co.string;
+  counter = co.number;
+}
```
Comment (gdorsi) [inline homepage/homepage/app/docs/[framework]/[...slug]/project-setup/svelte.mdx:29]: Small catch on the profile, the data can be read by everyone.

Maybe we should add a small comment about it?

Comment (bensleveritt) [reply homepage/homepage/app/docs/[framework]/[...slug]/project-setup/svelte.mdx:29]: Yeah I thought about and figured it's just a demo, but you're right a comment would help

Comment (bensleveritt) [reply homepage/homepage/app/docs/[framework]/[...slug]/project-setup/svelte.mdx:29]: Added a note

File: homepage/homepage/app/docs/[framework]/[...slug]/project-setup/svelte.mdx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+<CodeGroup>
+```sh
+pnpm install jazz-tools jazz-svelte
+```
+</CodeGroup>
+
+2. Write your schema
+
+See the [schema docs](/docs/react/schemas/covalues) for more information.
```
Comment (trishalim) [inline homepage/homepage/app/docs/[framework]/[...slug]/project-setup/svelte.mdx]: Typo here, you're linking to react docs

Comment (trishalim) [reply homepage/homepage/app/docs/[framework]/[...slug]/project-setup/svelte.mdx]: Other than that, all good.

**Pull request merged**